spacy/tests/pipeline/test_textcat.py

```diff
@@ -496,6 +496,38 @@ def test_textcat_evaluation():
     assert scores["cats_micro_r"] == 4 / 6
 
 
+def test_textcat_evaluation_missing():
+    train_examples = []
+    nlp = English()
+    ref1 = nlp("one")
+    # reference 'summer' is missing, pred 'summer' is 0
+    ref1.cats = {"winter": 1.0, "spring": 1.0, "autumn": 1.0}
+    pred1 = nlp("one")
+    pred1.cats = {"winter": 1.0, "summer": 0.0, "spring": 1.0, "autumn": 1.0}
+    train_examples.append(Example(pred1, ref1))
+
+    ref2 = nlp("two")
+    # reference 'spring' is missing, pred 'spring' is 1
+    ref2.cats = {"winter": 0.0, "summer": 0.0, "autumn": 1.0}
+    pred2 = nlp("two")
+    pred2.cats = {"winter": 1.0, "summer": 0.0, "spring": 1.0, "autumn": 1.0}
+    train_examples.append(Example(pred2, ref2))
+
+    scores = Scorer().score_cats(
+        train_examples, "cats", labels=["winter", "summer", "spring", "autumn"]
+    )
+    assert scores["cats_f_per_type"]["winter"]["p"] == 1 / 2
+    assert scores["cats_f_per_type"]["winter"]["r"] == 1 / 1
+    assert scores["cats_f_per_type"]["summer"]["p"] == 0
+    assert scores["cats_f_per_type"]["summer"]["r"] == 0 / 1
```

Comment:
```suggestion
    assert scores["cats_f_per_type"]["summer"]["r"] == 0
```
Nitpick: summer is never `True` in the reference annotation so recall is always 0 - removing the `/1` which seems to imply a missed prediction